lib/tasks/verify-origami-json.js

```diff
@@ -44,6 +44,13 @@ function origamiJson(config) {
 							if (hasExpanded) {
 								result.push('The expanded property has been deprecated. Use the "hidden" property when a demo should not appear in the Registry.');
 							}
+
+							const hasNoTitle = componentDemos.some(function (demo) {
+								return ! Object.prototype.hasOwnProperty.call(demo, 'title');
```

Comment:
This checks for existence of a property but not whether that property is a string nor whether it is an empty string.